command_lib/base.yml

```diff
@@ -87,3 +87,34 @@ debian:
           delimiter: 'EOF'
         src_urls: {}
 
+# openjdk -------------------------------------------------------------------------------------------------------------------------------
+openjdk:
+  latest: '8'
+  tags:
+    '8':
+        shell: '/bin/bash'
+        # either enter the package names as a list or a snippet to invoke either within or outside the container
+        names:
+          invoke:
+            1:
+              # Currently two types of environments are supported - host or container
+              container:
+                - "dpkg --get-selections | cut -f1 -d':' | awk '{print $1}'"
+          delimiter: "\n"
+        # the length of this list should be the same as the names or else there will be a mismatch
+        versions:
+          invoke:
+            1:
+              container:
+                - "pkgs=`dpkg --get-selections | cut -f1 -d':' | awk '{print $1}'`"
+                - "for p in $pkgs; do dpkg -l $p | awk 'NR>5 {print $3}'; done"
+          delimiter: "\n"
+        licenses:
+          invoke:
+            1:
+              container:
+                - "pkgs=`dpkg --get-selections | cut -f1 -d':' | awk '{print $1}'`"
+                - "for p in $pkgs; do cat /usr/share/doc/$p/copyright; echo EOF; done"
+          delimiter: 'EOF'
+        src_urls: {}
```

Comment:
Can you also change your commit message to the format below?

Add openjdk:8 to base.yml

openjdk:8 is based on Debian
<some more detail on the work you have done>

Signed-off-by: ....

Reply:
I have the changes you recommended completed and tested.  Would you like me to remove the last commit and add a new one or just go ahead and make a second commit?

Reply:
Please create a new one. Thanks!